test/new-e2e/tests/windows/command/agent.go

```diff
@@ -0,0 +1,36 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+package command
+
+const (
```

Comment:
Note for windows reviewers: had to duplicate this from https://github.com/DataDog/datadog-agent/blob/7b93707b1deffe12fde62b0040feedf00ee102af/test/new-e2e/tests/windows/common/agent/agent.go#L31 to avoid a cycle dependency, as `common` imports `clients` package

Reply:
maybe this could be shared somewhere like `windows/agent/` or `windows/const/`